ArduPlane/ArduPlane.cpp

```diff
@@ -67,7 +67,7 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(check_long_failsafe,     3,    400),
     SCHED_TASK(read_receiver_rssi,     10,    100),
     SCHED_TASK(rpm_update,             10,    100),
-    SCHED_TASK(airspeed_ratio_update,   1,    100),
```

Comment:
what's the reason for switching to 10Hz?

Reply:
one Hz was not entertaining enough :) I wanted to see what the noise looked like